src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -30,6 +31,8 @@
 /** Provides access to the data stored in Datastore. */
 public class Datastore {
 
+  static final int NUMBER = 1000;
```

Comment:
Sorry could you rename this variable to MESSAGE_LIMIT and write a comment.

/** The maximum number of message entries to fetch from Datastore. */
static final int MESSAGE_LIMIT = 1000;